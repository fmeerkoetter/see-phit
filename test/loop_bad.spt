R"*(
<for from=0 to=5>
  <div>
    item{{n}}
  </div>
</for>
)*"_html;
