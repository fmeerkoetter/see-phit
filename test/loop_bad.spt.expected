Error() [with int ROW = 2; int COL = 17; WHAT = spt::Invalid_syntax_in_for_tag]':
